ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Config.scala

```diff
@@ -27,6 +27,7 @@ final case class Config(
     shuffleParticipants: Boolean,
     partyAllocation: PartyAllocationConfiguration,
     ledgerClockGranularityMs: Int,
+    skipDarUpload: Boolean
```

Comment:
While we don't have the scalafmt rule for trailing commas turned on, I'd appreciate it if you add them for the sake of the next diff. :slightly_smiling_face:

```suggestion
    skipDarUpload: Boolean,
```

Reply:
I fixed the commas. I wanted to turn on trailing commas rule in scalafmt, but it's more complicated than it looks like. First, we need to bump version, secondly test result diff with lines that requires changes after that is 42000 lines long :scream: . I think I skip it for now ;-)

Reply:
Yes, please don't add a full scalafmt reformat into this PR ;)